products/billingbudget/terraform.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright 2019 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  Budget: !ruby/object:Overrides::Terraform::ResourceOverride
+    id_format: '{{name}}'
+    import_format: ['{{name}}']
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
```

Comment:
Looks like you added the code for the second example but it still needs to be added in this list